File: 4_Sorting_Algorithms/Bubble_Sort/Python/main.py
"""
This is my main module
"""
from bubble_sort import bubble_sort


def main():
    arr = [64, 34, 25, 12, 22, 11, 90]
    print("Original array: {}".format(arr))
    bubble_sort(arr)
    print("Sorted array: {}".format(arr))
    print('The array is sorted correctly.') if sorted(arr) == arr else print("The array is not sorted correctly.")


if __name__ == '__main__':
    main()

File: 4_Sorting_Algorithms/Bubble_Sort/C/main.c
#include "bubble_sort.h"

/**
 * main - This is the main Function
 * 
 * Return: 0
 */
int main(){
    int arr[] = {64, 34, 25, 12, 22, 11, 90};
    int arr_size = sizeof(arr)/sizeof(arr[0]);

    printf("Original array:");
    display_array(arr, arr_size);

    bubble_sort(arr , arr_size);

    printf("Sorted array:");
    display_array(arr, arr_size);
    is_sorted(arr, arr_size) ? printf("The array is sorted correctly.\n"): printf("The array is not sorted correctly.\n");

    return 0;
}